src/otx/tools/converter.py

```diff
@@ -438,6 +438,9 @@ def instantiate(
         model_parser.add_subclass_arguments(OTXModel, "model", required=False, fail_untyped=False, skip={"label_info"})
         model = model_parser.instantiate_classes(Namespace(model=model_config)).get("model")
 
+        if hasattr(model, "tile_config"):
+            model.tile_config = datamodule.tile_config
```

Comment:
Personally, I think this code should be inside Engine, what do you think?

Reply:
Should it be applied to OTX alone use case? If it's true, I also think that it should be implemented in Engine (or maybe CLI also).

Reply:
Yes, proceed like this for now, but I think it would be good to change it to a cleaner method later. This seems like a difficult structure to understand.